src/net_processing.cpp

```diff
@@ -391,6 +391,18 @@ struct Peer {
     /** Whether we've sent our peer a sendheaders message. **/
     std::atomic<bool> m_sent_sendheaders{false};
 
+    /** Length of current-streak of unconnecting headers announcements */
+    int m_unconnecting_headers GUARDED_BY(NetEventsInterface::g_msgproc_mutex){0};
```

Comment:
If this is changed, why not pick a better name, like `m_num_unconnecting_headers_msgs` (https://github.com/bitcoin/bitcoin/pull/25555/files)

Reply:
Right, makes sense. Also renamed `MAX_UNCONNECTING_HEADERS`.

Reply:
nit: I think MarcoFalke's suggestion `m_num_unconnecting_headers_msgs` (with `num`) is better than `m_unconnecting_headers_msgs`, quickly shows that this is a count.

Reply:
I meant to name it exactly like Marco did in #25555 🤦 Has the correct name now.